inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -31,6 +56,12 @@ class StaticShapeNMS final : public NonMaxSuppression {
                                   {DataType::S32}});
     }
 
+    void serializeParamsImpl(BlobSerializer& serializer) const override {
+        bool center_point_box = attrs().get<bool>("center_point_box");
```

Comment:
Nitpick: You can use `auto` here or even `const auto`

Reply:
Done.